### ISSUE TITLE
Awkward "Loading" Behavior in Gradebook Individual View

### ISSUE THREAD

**Issue by cycomachead** [instructure/canvas-lms#805]
### Summary:
When using the individual view page of a course's grade book, the "Graded Submissions: N" value under "Assignment Information" appears to do a lazy calculation. This means it's showing incorrect data to the user. 

(Sorry for the terse report - I'm in the midst of doing an analysis report for a course, which I encountered this bug. I'll update later.)

### Steps to reproduce:

 1.
 2.
 3.

### Additional notes:

**Comment by simonista** [comment]
Hey @cycomachead, I'd love to hear more details about how you're getting that to happen.  I can't reproduce it.  Here's what I'm doing:

- select an assignment
- do not select a student (so that the panel and the "next assignment" button are close enough together that I can see them both at the same time)
- hit "next assignment"
- the "graded submissions" count changes immediately.

**Comment by cycomachead** [comment]
Hi @simonista 

Sorry I realized I totally never got to this issue! 😞 

I wasn't doing anything particularly special. It was a UC Berkeley course with ~200 students.
I was on the individual view, and when I would click "Next assignment" I would immediately see a low number of graded assignments, and then things would slowly increase.
https://bcourses.berkeley.edu/courses/1408649/gradebook

I just checked again, and all seems to be well...I'm not sure what was up a few months ago.
